Sources/Misc/DangerousSettings.swift

```diff
@@ -58,6 +58,12 @@ import Foundation
      */
     @objc public let autoSyncPurchases: Bool
 
+    /**
+     * if `true`, the SDK will return a set of mock products instead of the
+     * products obtained from StoreKit. This is useful for testing or preview purposes. 
+     */
+    @_spi(Internal) public let uiPreviewMode: Bool
+
```

Comment:
maybe we can add the API tests to go along with it in this same PR? Those serve to also showcase the usage

Reply:
Done in f406e82a16500e5e6ecb9b2921ba31d1f3107c30